src/helpers/agGrid.ts

```diff
@@ -637,9 +653,16 @@ export const getGridIndex = (position: number, gridDistance: number) => {
 
 export const cellRenderer = (params: ICellRendererParams) => {
     const isRowTotal = params.data && params.data.type && params.data.type.rowTotal;
+
+    const isActiveRowTotal =
+        isRowTotal && // short circuit for non row totals
+        params.data &&
+        params.data.rowTotalActiveMeasures &&
+        params.data.rowTotalActiveMeasures.some((measure: string) => params.colDef.field.endsWith(measure));
```

Comment:
- measure: string is I think measureColId
- I would rather use getParsedFields(params.colDef.field).slice(-1) === measureColId which doesn't rely on code that needs a polyfill

Reply:
Ad 1) this makes sense, I updated the name.
Ad 2) the `getParsedFields` is not really handy here as it splits "too much". I would need to do something like

```js
getParsedFields(params.colDef.field).slice(-1).join(ID_SEPARATOR) === measureColId
```

which is kind of obscure. The polyfill will be handled by babel anyway. I'd rather keep the endsWith as it is clearer about what it is trying to do.